docs/resource/cicd.md

```diff
@@ -0,0 +1,191 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CICD
+--->
+
+# CICD
+
+<!-- toc -->
+
+- [Continuous Integration and Continuous Deployment (CICD)](#continuous-integration-and-continuous-deployment-cicd)
+  - [CICD pipeline resource attributes](#cicd-pipeline-resource-attributes)
+  - [CICD pipeline run resource attributes](#cicd-pipeline-run-resource-attributes)
+  - [CICD worker resource attributes](#cicd-worker-resource-attributes)
+- [Version Control System (VCS)](#version-control-system-vcs)
+  - [VCS repository resource attributes](#vcs-repository-resource-attributes)
+  - [VCS reference resource attributes](#vcs-reference-resource-attributes)
+
+<!-- tocstop -->
+
+See also:
+
+- The [Trace semantic conventions for CICD](/docs/cicd/cicd-spans.md)
+
+## Continuous Integration and Continuous Deployment (CICD)
+
+CICD aims to increase early defect discovery, increase productivity, and provide faster release cycles through automating builds, tests and deployment of software.
+
+### CICD pipeline resource attributes
+
+<!-- semconv resource.cicd.pipeline -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `cicd.pipeline`
+
+**Description:** A pipeline is a series of automated steps that helps software teams deliver code.
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`cicd.pipeline.name`](/docs/attributes-registry/cicd.md) | string | The human readable name of the pipeline within a CI/CD system. | `Build and Test`; `Lint`; `Deploy Go Project`; `deploy_to_environment` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### CICD pipeline run resource attributes
+
+<!-- semconv resource.cicd.pipelinerun -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `cicd.pipelinerun`
```

Comment:
```suggestion
**type:** `cicd.pipeline.run`
```

Reply:
Renamed in 77cca2d9657f7c4b02bfafdc6c444be5aafd7a30